backend/account/user.py

```diff
@@ -22,6 +22,11 @@ def create_user(self, email: str, user_id: str) -> User:
             raise error
         return user
 
+    def update_user(self, user: User, user_id: str) -> User:
+        user.user_id = user_id
+        user.save()
```

Comment:
Minor note, each `model_instance.save()` call would translate to a DB call - so in case multiple fields are edited in this function usage's context, it'll be good to do it directly

Reply:
Will look into optimization on different PR. I have noted it.